server/planning/assignments/assignments.py

```diff
@@ -336,6 +338,53 @@ def send_assignment_notification(self, updates, original=None, force=False):
         else:
             event_item = None
 
+        # Create the ICS file to be added to the email usable in google calendar.
+        ical = Calendar()
+        ical.add('BEGIN', 'VCALENDAR')
+        ical.add('PRODID', '-//Superdesk//NTB//EN')
+        ical.add('VERSION', '2.0')
+        ical.add('BEGIN', 'VEVENT')
+        ical.add('CLASS', 'PUBLIC')
+
+        if assignment['planning']['scheduled']:
```

Comment:
An Assignment should always have a scheduled date.
In any case, an Event inside an ical file is not usable if the Event does not have a start/end date/time